pkg/client/supplier/client.go

```diff
@@ -41,7 +41,7 @@ func NewSupplierClient(
 
 	if err := depinject.Inject(
 		deps,
-		&sClient.txClient,
+		&sClient.txClients,
```

Comment:
@red-0ne I know we talked about not trying to make depinject happy for now, but I think I'm stuck with this issue now:
```
can't resolve type github.com/pokt-network/poktroll/pkg/client/client.TxClientMap for github.com/pokt-network/poktroll/pkg/client/supplier.NewSupplierClient (/Users/dk/pocket/poktroll/pkg/client/supplier/client.go:42):
	while resolving:
		client.TxClientMap for github.com/pokt-network/poktroll/pkg/client/supplier.NewSupplierClient (/Users/dk/pocket/poktroll/pkg/client/supplier/client.go:42)
```

The supplier client should have the same issue (they are very look alike). What do you think I might have missed? Thanks!